### ISSUE TITLE
Get List Product Error Warning

### ISSUE THREAD

**Issue by EagleSH** [codeswholesale/codeswholesale-sdk-php#40]
sometimes we have this
Get List Product Error Warning: First parameter must either be an object or the name of an existing class in codeswholesale/sdk/src/CodesWholesale/Resource/Resource.php on line 48

and nothing works, why this?

**Comment by officialmmt** [comment]
It is actually working did you check the sample(list-products.php) at vendor?

**Comment by EagleSH** [comment]
> It is actually working did you check the sample(list-products.php) at vendor?

yes, its working now and as i said sometimes i get this error, today half of day i seeing this error. but now its suddenly started to work

any throughts?i  will be very appriciated

**Comment by EagleSH** [comment]
[2020-11-02 10:02:02] main.CRITICAL: Get List Product Error Server error: `GET https://api.codeswholesale.com/v2/products` resulted in a `504 Gateway Time-out` response:
<html>
<head><title>504 Gateway Time-out</title></head>
<body>
<center><h1>504 Gateway Time-out</h1></center>
</body (truncated...)

again

504 is gone but old one is come
main.CRITICAL: Get List Product Error Warning: First parameter must either be an object or the name of an existing class in codeswholesale/sdk/src/CodesWholesale/Resource/Resource.php on line 48 [] []

**Comment by mvdhoek1** [comment]
I've seen this error before. I use this package together with the Themosis framework and Wordpress. First I've tried to start an products import from the wp-cli. I guess this was to early in the proces so I was faced with the same error. Now I'm starting the import via my routes file and the error did not show up. 

Not sure if this answer is useful but maybe this can you give some sort of insight in to your problem.
